docs/en/architecture.md

```diff
@@ -0,0 +1,53 @@
+# Architecture
+gShell uses web technologies to make development easier! It runs on top of the Linux kernel. Here is a basic architecture overview (information on each section is below):
+
+- Linux Kernel
+  - X11 Xsession
+    - gShell (Electron runtime)
+      -  OOBS (Out-of-box setup)
+        - Basic power menu
+      - Lock screen
+        - Notifications
+        - Incoming call screen
+        - Basic power menu
+      - User environment
+        - User-specific power menu
+        - Home screen
+        - Notifications
+        - Incoming call screen
+        - App switcher
+          - Running privileged system apps
+            - (example) Settings
+            - (example) Sphere browser
+          - Running unprivileged apps
+            - (example) Calculator
+            - (example) Phone
+
+## Explanations
+**Linux Kernel** - The underlying system (Mobian in the case of the PinePhone build)
+
+**X11 Xsession** - Creates a graphical session to run gShell in
+
+**gShell (Electron runtime)** - The gShell runtime that uses Electron and Node.js. All child layers are represented in HTML.
+
+**OOBS (Out-of-box setup)** - When the gShell user uses their phone for the first time, this is shown, and it lets the user set up their phone, including options such as setting the passcode, the user's personal details (like their name), the phone's name, WiFi connection details etc.
+
+**Basic power menu** - A simple power menu that provides options to shutdown or restart the phone.
+
+**Lock screen** - A screen showing the current time above the user's wallpaper and notifications, amongst other things. It also checks the password before unlocking the phone and switching to the user environment.
+
+**Notifications** - The notifications drawer that contains various settings (some of which are only accessible inside the user environment).
+
+**User environment** - The environment used by the user once they have unlocked their phone, and lets the user open apps and perform other functions.
+
+**User-specific power menu** - a power menu that extends the basic power menu to include shortcuts to time-critical apps (e.g. to show the user's bus pass or pay for an item)
+
+**Home screen** - Lets the user launch apps. Its appearance can be customised (such as wallpaper, icons, and icon placement)
+
+**Incoming call screen** - When somebody is calling, this screen pops up to let the user answer the call, and then it handles the UI throughout the duration of the call.
+
+**App switcher** - Lets the user switch between running apps, and contains the running apps as child elements inside ```webview``` tags.
```

Comment:
Overall looks good to me! For the inline code blocks I'd use one backtick instead of three, but seems that it's still well supported either way.

Reply:
Will merge now!